src/main.js

```diff
@@ -1 +1,53 @@
-console.log('Hello, world!');
+// something cool i learned :
+// before i chose mousedown n click as an event so if i click on the value the chosen input changes but the function doesnt work because the event is mouse down it must exactly be click on the radio button..... 
```

Comment:
Interesting...For me, mousedown only triggers when I click the actual round button, but click triggers when clicked on the whole `<label>` element.